storage/cloud-client/public_access_prevention_test.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+import uuid
+
+from google.cloud import storage
+import pytest
+
+import storage_enforce_public_access_prevention
+import storage_get_public_access_prevention
+import storage_unspecify_public_access_prevention
+
+
+@pytest.fixture()
+def bucket():
```

Comment:
Is this something that could/should be shared across sample tests that use buckets?

Reply:
It could be shared. I'm not sure it should be. I based these tests off of the UBLA tests since these are similar features and the UBLA tests do not share the bucket.

Reply:
Oh I didn't mean that the bucket should be shared, but rather the fixture code could be. Does that make sense?

Reply:
Ah I understand now. I will look into it!